prow/repoowners/repoowners.go

```diff
@@ -736,8 +737,35 @@ func (o *RepoOwners) applyOptionsToPath(path string, opts dirOptions) {
 	}
 }
 
-func (o *RepoOwners) filterCollaborators(toKeep []github.User) *RepoOwners {
-	collabs := sets.New[string]()
+// Extend github team to appproves/reviewers list
+// If appproves/reviewers name contains '/', like 'myorg/developer'
+// It will be recognized as github team.
```

Comment:
pls either fixup the godoc so that it follows the established `<name> does this and this` pattern or remove it